formulas/functions/statistical.js

```diff
@@ -251,8 +251,30 @@ const StatisticalFunctions = {
 
   MIN: () => {},
 
-  MINA: () => {},
-
+  MINA: (...values) => {
+    const arr = values.map(a => a.value)
+    const flatArr = FormulaHelpers.flattenDeep(arr)
+    const updatedArr = flatArr.map(a => {
+      if(a === true)
+        return 1
+      if(isNaN(a - a))
+        return 0
+      if(!isNaN(parseFloat(a)))
+        return parseFloat(a)
+      if(!isNaN(parseInt(a)))
+        return parseInt(a)
+      if(["boolean", "string"].includes(typeof a))
+        return 0
+      if(typeof a === "number")
+        return a
+      return null
```

Comment:
Can probably just remove the middle three cases to match behavior of other spreadsheets (which do not try to parse strings).

The `isNaN(a - a)` is a nice way to identify most strings as well as `Infinity` and `NaN`, but should probably come after the string filter (so it is _only_ handling Infinity and NaN) ... and should maybe come with a comment, since it's not obvious what it's doing.

Consider grouping cases by input type rather than response, so:

```suggestion
      if(typeof a === "boolean")
        return +a
      if(typeof a === "string")
        return 0
      if(isNaN(a - a))
        return 0 
      if(typeof a === "number")
        return a
      return 0
```